### PR TITLE
LITTIL-35 Cookies

### PR DESCRIPTION
**Pull request by MarcelWildenburg**

### CODE REVIEW

**Review by kappy904 — APPROVED** [review]
Would be nice to include a staging link or screenshot for visual review as well

File: src/app/app.component.spec.ts [hunk 1]
```diff
@@ -9,6 +9,7 @@ import { MainMenuButtonComponent } from './components/main-menu-button/main-menu
 import { MainMenuDropdownButtonComponent } from './components/main-menu-dropdown-button/main-menu-dropdown-button.component';
 import {UserMenuComponent} from "./components/user-menu/user-menu.component";
 import { PermissionController } from './services/permission.controller';
+import {NgcCookieConsentService} from "ngx-cookieconsent";
```
[the remark this answers is not in the document]
Comment (MarcelWildenburg) [reply src/app/app.component.spec.ts:12]: image

File: angular.json [hunk 2]
```diff
@@ -27,8 +27,8 @@
             "tsConfig": "tsconfig.app.json",
             "inlineStyleLanguage": "scss",
             "assets": ["src/assets", "src/config.js"],
-            "styles": ["src/styles.scss", "./node_modules/leaflet/dist/leaflet.css"],
-            "scripts": []
+            "styles": ["src/styles.scss", "./node_modules/leaflet/dist/leaflet.css", "./node_modules/cookieconsent/build/cookieconsent.min.css"],
```
[the remark this answers is not in the document]
Comment (MarcelWildenburg) [reply angular.json]: fixed

File: src/app/app.component.spec.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarcelWildenburg) [reply src/app/app.component.spec.ts]: fixed

**Pull request merged**